src/gawronski-bartosz/Fsd.Bartek.Cs/Fsd.Bartek.Cs.Ex2/Domain/UI/ProgramUi.cs

```diff
@@ -18,16 +18,16 @@ public static void InsertNumber()
                 switch (item)
                 {
                     case "cats":
-                        Producent.FabricCat(int.Parse(number));
+                        Producent.FabricAnimal(int.Parse(number),item);
                         break;
                     case "dogs":
-                        Producent.FabricDog(int.Parse(number));
+                        Producent.FabricAnimal(int.Parse(number),item);
```

Comment:
brak formatowania (odstępy). Użyj VS żeby zrobił to za Ciebie. Poza tym FabricAnimal z niczym mi się nei kojarzy.